src/mongoChangeStream.test.ts

```diff
@@ -127,36 +127,36 @@ const initState = async (sync: SyncObj, db: Db, coll: Collection) => {
 }
 
 describe('syncing', () => {
-  test('starting change stream is idempotent', async () => {
+  // NOTE: This test is flaky
```

Comment:
Just curious, what's flaky about it? I don't see any assertions. Does it sometimes throw an error?

Reply:
I don't fully understand why after spending around 30 minutes looking into it.